ONSdigital/census-rh-ui
# CR 1183 CE Add Room Number

# Motivation and Context
Allow users requesting a CE fulfilment to enter a room number in addition the selected address

# What has changed
Updates to CE fulfilments, to allow the user on the 'Confirm Address' screen, and/or the 'Confirm Name and Address' screen to specify a 'room number'. 
If added at either point the room number value is either appended to the end of the surname, or if the surname is too long is provided as the optional 'title' value in the RHSvc request fulfilment endpoint
Unit tests added to test for the journey additions, and to ensure values and links are not present in non-CE journeys.

No Cucumber update is required to merge

# How to test?
Request a UAC or paper form fulfilment, and enter a CE as the address. Ensure the 'Add room number' link appears on the address confirmation screen, can be entered, and then appears on the address confirmation screen. Repeat, but set the value in the 'Confirm Name and Address' screen. When set it should be possible to change the value on both screens.
Journey should complete as normal, with the room number appended in confirm address displays, and at the sent confirmation screens

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1183

# Screenshots (if appropriate):
